site/content/en/docs/drivers/includes/docker_usage.inc

```diff
@@ -1,6 +1,6 @@
 ## Requirements
 
-- [Install Docker](https://hub.docker.com/search?q=&type=edition&offering=community&sort=updated_at&order=desc)
+- [Install Docker](https://hub.docker.com/search?q=&type=edition&offering=community&sort=updated_at&order=desc) 18.07 or higher
```

Comment:
Docker 18.07 does not exist, I think you meant 18.09 ?

See https://docs.docker.com/engine/release-notes/

Reply:
I agree, lets go with 18.09

Reply:
Fix both of code and doc to 18.09